cmd/triggerRun/cmd/root.go

```diff
@@ -43,10 +53,55 @@ var (
 func init() {
 	rootCmd.Flags().StringVarP(&triggerFile, "triggerFile", "t", "", "Path to trigger yaml file")
 	rootCmd.Flags().StringVarP(&httpPath, "httpPath", "r", "", "Path to body event")
+	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "absolute path to the kubeconfig file")
 }
 
 func rootRun(cmd *cobra.Command, args []string) {
```

Comment:
we should not hardcode these but rather use the values from the flags above i.e. triggerFile and httpPath vars.